packages/adapter/addon/rest.js

```diff
@@ -299,7 +302,7 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
     // Avoid computed property override deprecation in fastboot as suggested by:
     // https://deprecations.emberjs.com/v3.x/#toc_computed-property-override
     get() {
-      if (this._fastboot) {
+      if (this._fastboot !== undefined) {
```

Comment:
since _fastboot is not a boolean (whereas _useFetch is) we don't need to add the undefined check for it, presence check is enough